src/yank_config.rs

```diff
@@ -432,10 +431,7 @@ pub async fn yank_config() -> Result<(), YankError> {
     config::check_archetype_name_matches(&config).map_err(|e| YankError::ArchetypeNameError(e))?;
 
     std::fs::write(
-        std::path::Path::new(env!("CARGO_MANIFEST_DIR"))
-            .parent()
-            .unwrap()
-            .join("hcor/config/content.json"),
+        std::path::Path::new("../hcor/config/content.json"),
```

Comment:
You should be able to omit `std::path::Path::new` here and just supply the string, since `std::fs::write` takes `AsRef<Path>`.